### PR TITLE
MAGEDOC-4024 New Cloud topic for managing disk space

### PR DESCRIPTION
**Pull request by hguthrie**
This PR addresses the conversion of two KB topics into a new topic for the Manage your project section of the Cloud guide.

whatsnew
Added the [Manage disk space topic](https://devdocs.magento.com/guides/v2.3/cloud/project/manage-disk-space.html) to the Cloud guide.

### CODE REVIEW

File: guides/v2.2/cloud/project/manage-disk-space.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+group: cloud-guide
+title: Manage disk space
+functional_areas:
+  - Cloud
+  - Storage
+---
+You can find the total storage capacity for your Cloud project in your {{site.data.var.ece}} contract and in your [Magento account page](https://accounts.magento.cloud/user). Each project card in your account shows the number of _environments_, the _storage_ capacity in GB, and the number of _users_.
```
Comment (meker12) [inline guides/v2.2/cloud/project/manage-disk-space.md]:
```suggestion
You can find the total storage capacity for your Cloud project in your {{site.data.var.ece}} contract and on your [Magento account page](https://accounts.magento.cloud/user). Each project card in your account shows the number of _environments_, the _storage_ capacity in GB, and the number of _users_.
```

File: guides/v2.2/cloud/project/manage-disk-space.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+You can check disk space usage in each environment using the _disk free_ command, which reports the amount of disk space used by the file system. You must use SSH to log in to a remote environment.
```
Comment (meker12) [inline guides/v2.2/cloud/project/manage-disk-space.md]:
```suggestion
You can check disk space usage in each environment using the _disk free_ (`df`) command, which reports the amount of disk space used by the file system. You must use SSH to log in to a remote environment.
```

File: guides/v2.2/cloud/project/manage-disk-space.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+```
+
+Sample response:
+
+```terminal
+Filesystem                     Size  Used Avail Use% Mounted on
+/dev/mapper/35xdpijwfe         2.0G  1.9G     0 100% /app/var
+```
+
+For the database, you can use the CLI to check approximate disk space usage:
```
Comment (meker12) [inline guides/v2.2/cloud/project/manage-disk-space.md]:
```suggestion
For the database, you can use the {{ site.data.var.ece }} CLI to check approximate disk space usage:
```

File: guides/v2.2/cloud/project/manage-disk-space.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+1. Save changes in the file.
+
+1. Add, commit, and push your code changes.
+
+   ```bash
+   git add -A && git commit -m "Increase disk space for application" && git push magento <branch-name>
+   ```
+
+   The changes come into effect after you push the updated YAML file to the remote environment.
```
Comment (meker12) [inline guides/v2.2/cloud/project/manage-disk-space.md]:
```suggestion
   The changes take effect after you push the updated YAML file to the remote environment.
```

File: guides/v2.2/cloud/project/manage-disk-space.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+1. Save changes in the file.
+
+1. Add, commit, and push your code changes.
+
+   ```bash
+   git add -A && git commit -m "Increase disk space for service" && git push magento <branch-name>
+   ```
+
+   The changes come into effect after you push the updated YAML file to the remote environment.
```
Comment (meker12) [inline guides/v2.2/cloud/project/manage-disk-space.md]:
```suggestion
   The changes take effect after you push the updated YAML file to the remote environment.
```

File: guides/v2.2/cloud/project/manage-disk-space.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
++----------------+-----------------+--------+
+| Allocated disk | Estimated usage | % used |
++----------------+-----------------+--------+
+| 2.0 GiB        | 193.3 MiB       | ~ 9%   |
++----------------+-----------------+--------+
+```
+
+## Allocating disk space
+
+There are two configuration files that control the allocation of disk space in the Integration environment: the `.magento.app.yaml` file and the `.magento/services.yaml` file. Each file contains the `disk` property used to define the disk size value in MB for the respective configuration.
```
Comment (meker12) [inline guides/v2.2/cloud/project/manage-disk-space.md]:
```suggestion
Two configuration files control the allocation of disk space in the Integration environment–the `.magento.app.yaml` file and the `.magento/services.yaml` file. Each file contains the `disk` property which specifies the disk size value in MB for the respective configuration.
```

**Comment by hguthrie** [comment]
running tests

**Pull request merged**